src/ServicePulse.Host/app/js/directives/ui.particular.largeGraph.js

```diff
@@ -0,0 +1,113 @@
+﻿(function(window, angular, undefined) {
+    'use strict';
+
+    angular.module('ui.particular.largeGraph', [])
+        .directive('largeGraph',
+            function() {
+                return {
+                    restrict: 'E',
+                    scope: {
+                        data: '=plotPoints',
+                        avg: '=plotAverage',
+                        dates: '=xaxisPoints',
+                        width: '=plotWidth',
+                        heigth: '=plotHeight'
+                    },
+                    template: '<svg></svg>',
+                    link: function link(scope, element, attrs) {
+                        scope.$watch('data', function () {
```

Comment:
Is the watch really necessary? Does it not rebind when the entire `throughput` object is rebound?

Reply:
It is necessary. The reason why it is not needed in other places is due to the fact that it is displayed in grid where we change whole collection => all html is rerendered. In here the control stays, and it was not refreshing to data changes. I verified and that was the suggested solution.